etl/open311/open311_to_socrata.py

```diff
@@ -0,0 +1,169 @@
+"""
+Checks open311 API for new CSRs and updates a Socrata Dataset
+"""
+
+import argparse
+from datetime import datetime, timedelta, timezone
+from zoneinfo import ZoneInfo
+import os
+import logging
+import requests
+import time
+
+from sodapy import Socrata
+
+from etl import utils
+
+# Socrata Secrets
+SO_WEB = os.getenv("SO_WEB")
+SO_TOKEN = os.getenv("SO_TOKEN")
+SO_KEY = os.getenv("SO_KEY")
+SO_SECRET = os.getenv("SO_SECRET")
+REALTIME_DATASET = os.getenv("REALTIME_DATASET")
+
+# Open 311 API Key
+API_KEY = os.getenv("OPEN_311_API_KEY")
+API_BASE_URL = os.getenv("OPEN_311_API_BASE_URL")
+
+headers = {"Authorization": f"Bearer {API_KEY}"}
+
+# 10 requests per minute API limit
+MAX_REQUESTS_PER_MINUTE = 10
```

Comment:
Bill from 311 is in contact with the vendor to see if we can get a higher limit but 1000 records per minute is more than enough for us if this is run every 5 mins or so.